tidy3d/components/medium.py

```diff
@@ -5162,8 +5168,8 @@ def get_background(comp: Axis) -> PoleResidue:
             meds = [get_component(med=med, comp=comp) for med in adjacent_media]
             # the Yee site for the E field in the normal direction is fully contained
             # in the medium on the + side
-            if comp == axis:
-                return meds[1]
+            # if comp == axis:
```

Comment:
By commented this out the normal component will also be computed as a weighted average